src/Sylius/Bundle/ApiBundle/DataTransformer/OrderItemIdAwareInputCommandDataTransformer.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\DataTransformer;
+
+use Sylius\Bundle\ApiBundle\Command\OrderItemIdAwareInterface;
+
+class OrderItemIdAwareInputCommandDataTransformer implements CommandDataTransformerInterface
+{
+    public function transform($object, string $to, array $context = [])
+    {
+        $object->setOrderItemId($this->getOrderItemIdFromUri($context['request_uri']));
```

Comment:
I wonder if you can get some more information from context to choose this id in a better way instead of getting the last part of IRI. The current solution works for the implemented endpoint, but is not a generic one, what could be problematic in future.

Reply:
Unfortunatelly there are 3 places where there is ID 1st one was this, 2nd was full request ( with hostname) and 3rd place were orderItemId's that were in order ( but this is not something we could use) hence I choose this one